LeMinhQuoc/blog-frontend
# Feature/detail blog

- Solve : display detailBlog
- How dose it PR solve it
   + get content blog from api/blogs/id when use click in a blog title on home page 
   + render to detail/index.twig
   + show content of a blog
- How can reviewer test the changes
   + Pull this branch on your local.
   + Install API source
   + Run composer install.
   + Run symfony server:start
   + Flow https://localhost/  and click a blog's title to check
